### PR TITLE
🌱 go.mod: Update github.com/google/go-github to v48.2.0

### PR DESCRIPTION
**Pull request by cpanato**
**What this PR does / why we need it**:

🌱 go.mod: Update github.com/google/go-github to v48.2.0

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

/assign @fabriziopandini

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -150,6 +149,8 @@ require (
 	sigs.k8s.io/structured-merge-diff/v4 v4.2.3 // indirect
 )
 
+require github.com/google/go-github/v48 v48.2.0
```
Comment (sbueringer) [inline go.mod:152]: Let's please change the version directly inline in l.20

**Review by sbueringer — COMMENTED** [review]
Thx!

one nit otherwise lgtm

I'm slightly confused why dependabot didn't suggest the bump

File: go.mod (hunk 1 above)
Comment (cpanato) [reply go.mod:152]: dependabot does not update this version on the left side

Comment (cpanato) [reply go.mod]: done, PTAL @sbueringer

Comment (sbueringer) [reply go.mod]: Ah good point.

**Comment by sbueringer** [comment]
/lgtm
Thx!

**Review by killianmuldoon — APPROVED** [review]
/lgtm

**Pull request merged**